xml/Microsoft.VisualBasic/VariantType.xml

```diff
@@ -505,7 +505,7 @@
       <MemberValue>4</MemberValue>
       <Docs>
         <summary>
-          <see langword="Single" />. (-3.402823E+38 through -1.401298E-45 for negative values; 1.401298E-45 through 3.402823E+38 for positive values.) This member is equivalent to the Visual Basic constant <see langword="vbSingle" />.</summary>
+          <see cref="T:System.Single" />. (-3.402823E+38 through -1.401298E-45 for negative values; 1.401298E-45 through 3.402823E+38 for positive values.) This member is equivalent to the Visual Basic constant <see langword="vbSingle" />.</summary>
```

Comment:
All the values in this enum type seem to be describing another type and should also be fixed. For example, in line 480 of this same file, langword=Short" should be cref=T:System.Short".

Reply:
These seem to be referring to VB types rather than .NET types. Can you confirm?

Reply:
The enum summary is saying these are types, but I'm seeing also Null. I don't know if that one is treated as a type in VB or a keyword.
I say we exclude this file from the PR. Can you please revert it?

Reply:
If you can confirm that the APIs described in each enum's descriptions using langword are actually types, feel free to change them to crefs.

Reply:
Reverted. These are actually VB keywords, although they don't show up in `<see langword="" />` completion list.